docs/guides/simulated_device_linux.md

```diff
@@ -1,100 +1,76 @@
 
 In order to utilize the app against a commissioner or controller, the app will
 need to be specifically built.
 
-1. To generate the ZAP files, and build the `chip-app1` binary completing the
+1. To build the `chip-app1` binary completing the
    following steps:
 
-    ```
-     ./scripts/examples/gn_build_test_example.sh app1
-    ```
+   ```
+   ./scripts/examples/gn_build_example.sh examples/placeholder/linux out/debug/simulated/ chip_tests_zap_config=\"app1\"
+   ```
 
-## Build the App only
+## Build the App with gn and ninja
 
 In order to utilize the app against a commissioner or controller, the app will
 need to be specifically built.
 
 1. To only build the `chip-app1` binary completing the following steps:
 
-    ```
-    source scripts/activate.sh
-    CHIP_ROOT="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
-    GN_ARGS="chip_tests_zap_config=\"app1\""
-    GN_ARGS+="chip_project_config_include_dirs=[\"$CHIP_ROOT/examples/placeholder/linux/apps/app1/include\", \"$CHIP_ROOT/config/standalone\"]"
-    GN_ARGS+="chip_config_network_layer_ble=false"
-
-    gn gen --check --fail-on-unused-args --root=examples/placeholder/linux out/simulated --args="$GN_ARGS"
-
-    ninja -C out/simulated
-    ```
+   ```
+   source scripts/activate.sh
+   gn gen --check --root=examples/placeholder/linux out/simulated --args="chip_tests_zap_config=\"app1\""
+   ninja -C out/simulated
+   ```
 
 ## Running the app
 
 Now that the building is completed there is a `chip-app1` binary created. This
 binary can be executed on a linux os.
 
-1. To generate the ZAP files, and build the `chip-app1` binary completing the
-   following steps:
-
-    ```
-    ./out/simulated/chip-app1
-    ```
+   ```
+   ./out/simulated/chip-app1
+   ```
 
 ## Running the app with test parameter
 
 Now that the building is completed there is a `chip-app1` binary created. This
 binary can be executed on a linux os with test commands.
 
-1. To generate the ZAP files, and build the `chip-app1` binary completing the
-   following steps:
-
-    ```
-    ./out/simulated/chip-app1 --command [TEST NAME]
-    ```
+   ```
+   ./out/simulated/chip-app1 --command [TEST NAME]
+   ```
 
 ## Interacting with the simulated app
 
 Now that the building the app and starting it is complete, you will be able to
 interact with it using chip-tool
 
-1. Follow the instruction to build chip-tool in the
-   [chip-tool readme](../../examples/chip-tool/README.md).
+1. Follow the instruction to build chip-tool in the [chip-tool readme](../../examples/chip-tool/README.md).
 
-2. Run this command to commission.
-    ```
-    ./out/debug/standalone/chip-tool pairing code 0x654321 MT:-24J0AFN00KA0648G00
-    ```
-    or whatever is listed on the "SetupQRCode:" line in the log output.
-3. Most tests will start at this point and now an send cluster commands with
-   chip-tool as follow.
+2. Run this command to commission with whatever is listed on the "SetupQRCode:" line in the log output.. 
```

Comment:
This is missing the actual command to run and also has an extra period at the end

Reply:
Thank you